### PR TITLE
refactored gerber and excellon drill into circuit-json-to-gerber

### PR DESCRIPTION
**Pull request by ShiboSoftwareDev**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -3,8 +3,6 @@ export * from "./lib/builder"
 export * from "./lib/apply-selector"
 export * from "./lib/types"
 export * from "./lib/utils"
-export * from "./lib/gerber"
-export * from "./lib/excellon-drill"
```
Comment (seveibar) [inline src/index.ts:7]: can we add `export * from "circuit-json-to-gerber"` for backwards compatibility? It'll break less things

**Review by seveibar — APPROVED** [review]
nice, make sure to get the test suite to pass before merging, nice refactor!

File: src/index.ts (hunk 1 above)
Comment (ShiboSoftwareDev) [reply src/index.ts:7]: ok

**Comment by ShiboSoftwareDev** [comment]
@seveibar why is this format bot failing?

**Comment by seveibar** [comment]
hmm i think it's using an old broken version of formatbot, can you update the formatbot workflow with the latest one from [tscircuit/plop](https://github.com/tscircuit/plop)

**Comment by ShiboSoftwareDev** [comment]
@seveibar the new one is expecting biome should I format files with biome?

**Comment by seveibar** [comment]
OOPS i forgot this repo used prettier- no we shouldn't do a full format right now- lets just remove that workflow for now. We can format this repo in a future pr

**Pull request merged**